src/lib.rs

```diff
@@ -17,12 +17,12 @@
 
 #![doc(html_root_url = "https://hyperium.github.io/mime.rs")]
 #![cfg_attr(test, deny(warnings))]
-#![cfg_attr(all(features = "nightly", test), feature(test))]
+#![cfg_attr(all(feature = "nightly", test), feature(test))]
 
 #[macro_use]
 extern crate log;
 
-#[cfg(features = "nightly")]
+#[cfg(feature = "nightly")]
```

Comment:
If it passes travis, then yes :D. Since travis is currently only setup to run with beta for this repo.